yt_dlp/extractor/viki.py

```diff
@@ -295,15 +309,35 @@ def _real_extract(self, url):
                 'id': thumbnail_id,
                 'url': thumbnail.get('url'),
             })
+        
+        stream_url = 'playback_streams/%s.json?drms=dt1,dt2&device_id=%s&token=%s' % (video_id, self._DEVICE_ID, self._token)
+        resp = self._call_api(stream_url, video_id, 'Downloading video streams JSON')
 
+        stream_id = resp['main'][0]['properties']['track']['stream_id']
```

Comment:
Use `try_get`. If subs cant be found, the rest of the extraction should still run error-free

Reply:
it's always present it the response.

Reply:
Don't jinx it, it's been seen there's no such thing as absolute certainty in websites.